2696. Minimum String Length After Removing Substrings/solution.py

```diff
@@ -9,3 +9,5 @@
 		# in both loops, if a match is found, cut those two characters out of the string and reform as a new string
```

Comment:
This is just a heads up that you need to be careful modifying a collection while iterating through it. So if you use an iterator like "for letter in s" -> this creates an iterator for s and modifying s underneath is potentially dangerous and a bad practice. In these cases, you want to loop using the index, so something like "for i in range(len(s))" or "while i < len(s))

Reply:
I don't think "for i in range(len(s))" is okay. I think len(s) is prematurely defined, so you'll have an index error pretty soon.

Reply:
I know the while loop works and I think you should use that Owen.

Reply:
oh I didn't know about modifying s underneath being bad practice! I will test out the index vs. while loop while implementing.